scripts/rff/test_data/output/WeatherVariability_Counties.tmcf

```diff
@@ -0,0 +1,89 @@
+
+    Node: E:WeatherVariability_Counties->E0
```

Comment:
With the earlier comment, we can get rid of leading whitespaces.  Also, we can drop the "output" files.

Instead might be a good idea to helpful to expected and compare it as well.